examples/tiny_example.py

```diff
@@ -34,6 +35,18 @@ def export_records():
                                           file_name="export_data")
 
 
+@app.route("/download_unicode_file_name_utf8", methods=['GET'])
```

Comment:
I see that you would like to show both str and unicode are handled with your change. However, I would suggest let's keep one of the two demo functions.